brage-import/src/test/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumerTest.java

```diff
@@ -1177,6 +1178,60 @@ void shouldMigrateHiddenFileFromIncomingWhenExistingPublicationDoesNotHaveHidden
                        .anyMatch(HiddenFile.class::isInstance));
     }
 
+    @Test
+    void shouldPersistNewFileWithUserInstanceFromIncomingPublicationWhenMergingPublications()
+        throws IOException, InvalidUnconfirmedSeriesException {
```

Comment:
This huge test is disgusting. But it is the only to merge two publication via handler and nothing can be extracted.